src/views/QueryEditor.tsx

```diff
@@ -37,57 +57,69 @@ const queryEditors: {
   },
   [QueryType.Issues]: {
     component: (props: Props, onChange: (val: any) => void) => (
-      <QueryEditorIssues {...(props.query.issuesOptions || {})} onChange={onChange} datasource={props.datasource} />
+      <QueryEditorIssues {...(props.query.issuesOptions || {})} onChange={onChange} datasource={props.datasource} owner={props.query.owner || ''} repository={props.query.repository || ''}/>
     ),
     optionsKey: 'issuesOptions',
   },
-  [QueryType.PullRequests]: {
+  [QueryType.Pull_Requests]: {
     component: (props: Props, onChange: (val: any) => void) => (
       <QueryEditorPullRequests {...(props.query.pullRequestsOptions || {})} onChange={onChange} />
     ),
     optionsKey: 'pullRequestsOptions',
   },
 };
 
-export const LeftColumnWidth = 12;
-export const RightColumnWidth = 36;
-
-const queryTypeOptions: Array<SelectableValue<QueryType>> = Object.keys(QueryType)
-  .filter((_, i) => QueryType[i] !== undefined)
-  .map((_, i) => {
+const queryTypeOptions: Array<SelectableValue<string>> = Object.keys(QueryType)
+  .map((v) => {
     return {
-      label: `${QueryType[i]}`,
-      value: i as QueryType,
+      label: v.replace("_", " "),
```

Comment:
can there be multiple underscores in the value?  this won't replace them all

Reply:
yeah. Is there an optional argument I missed?

Reply:
```
v.replace(/_/g, " ");
```

Reply:
updated it to a regex as suggested here:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

thanks for the callout I did not know that at all.